lib.php

```diff
@@ -45,18 +45,29 @@
  */
 function hvp_supports($feature) {
     switch($feature) {
-        case FEATURE_GROUPS:                  return true;
-        case FEATURE_GROUPINGS:               return true;
-        case FEATURE_GROUPMEMBERSONLY:        return true;
-        case FEATURE_MOD_INTRO:               return false;
-        case FEATURE_COMPLETION_TRACKS_VIEWS: return false;
-        case FEATURE_COMPLETION_HAS_RULES:    return false;
-        case FEATURE_GRADE_HAS_GRADE:         return true;
-        case FEATURE_GRADE_OUTCOMES:          return false;
-        case FEATURE_BACKUP_MOODLE2:          return false;
-        case FEATURE_SHOW_DESCRIPTION:        return false;
-
-        default: return null;
+        case FEATURE_GROUPS:
+return true;
+        case FEATURE_GROUPINGS:
+return true;
+        case FEATURE_GROUPMEMBERSONLY:
+return true;
+        case FEATURE_MOD_INTRO:
+return false;
+        case FEATURE_COMPLETION_TRACKS_VIEWS:
+return false;
+        case FEATURE_COMPLETION_HAS_RULES:
+return false;
+        case FEATURE_GRADE_HAS_GRADE:
+return true;
+        case FEATURE_GRADE_OUTCOMES:
+return false;
+        case FEATURE_BACKUP_MOODLE2:
+return false;
+        case FEATURE_SHOW_DESCRIPTION:
+return false;
+
+        default:
+return null;
```

Comment:
This one is odd indeed... The error from the Moodle codechecker is:
"Closing brace must be on a line by itself"
but how it fixed it is obviously wrong.

Let me look into this a bit more.

On Thu, Mar 31, 2016 at 3:40 AM Frode Petterson <notifications@github.com>
wrote:

> In lib.php
> <https://github.com/h5p/h5p-moodle-plugin/pull/42#discussion_r58011930>:
>
> > +return false;
> > +        case FEATURE_COMPLETION_TRACKS_VIEWS:
> > +return false;
> > +        case FEATURE_COMPLETION_HAS_RULES:
> > +return false;
> > +        case FEATURE_GRADE_HAS_GRADE:
> > +return true;
> > +        case FEATURE_GRADE_OUTCOMES:
> > +return false;
> > +        case FEATURE_BACKUP_MOODLE2:
> > +return false;
> > +        case FEATURE_SHOW_DESCRIPTION:
> > +return false;
> > +
> > +        default:
> > +return null;
>
> Did something go wrong here? It seems a bit strange that there should be
> no indentation.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/h5p/h5p-moodle-plugin/pull/42/files/6e6b5937ac8face5ffbc358caf801bd9a9b19040..c99ca73dd12e55d783b9b4e2225853aecce82b66#r58011930>
>
-- 

Mike Churchward | Executive Director | POET
mike.churchward@poetgroup.org
1.519.301.2287
http://poetgroup.org